settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
 
 INSTALLED_APPS = [
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
     "db",
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` setting is correctly set to `db.User`. Ensure this is consistent across your project to avoid conflicts with Django's default user model.